### PR TITLE
feat: add age formatter

### PR DESCRIPTION
**Pull request by seb-jean**
I add `age` formatter

Examples:
```twig
{# with filter: #}
Age: {{ user.birthdate|age }} <!-- Age: 30 years old -->

{# ... or use the equivalent function: #}
Age: {{ age(user.birthdate) }} <!-- Age: 30 years old -->
```

### CODE REVIEW

File: translations/time.en.xliff [hunk 1]
```diff
@@ -74,6 +74,10 @@
                 <source>duration.none</source>
                 <target>&lt; 1 second</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>age</source>
+                <target>1 year old|%count% year old</target>
```
Comment (kbond) [inline translations/time.en.xliff:79]:
```suggestion
                <target>1 year old|%count% years old</target>
```

?

**Review by weaverryan — COMMENTED** [review]
Hi!

This is a nice PR - but does it solve a complex enough problem to be in the library? I'm not sure...

**Comment by weaverryan** [comment]
Ha! Ok, now I see #190 - (I'm reading my notifications in reverse). Ok then - could you rebase for the translation conflict?

**Comment by seb-jean** [comment]
> Ha! Ok, now I see #190 - (I'm reading my notifications in reverse). Ok then - could you rebase for the translation conflict?

@weaverryan, no problem 😄 ! It is resolved.

**Comment by weaverryan** [comment]
Thanks @seb-jean! Now we have yet MORE translations to collect from languages /cc @kbond

**Comment by seb-jean** [comment]
Hi @weaverryan and @kbond ,
Do you think it is possible to publish a new release with `age` formatter.
Thank you :)

**Pull request merged**